airflow/utils/decorators.py

```diff
@@ -64,7 +64,7 @@ def wrapper(*args, **kwargs):
             dag_args = copy(dag.default_args) or {}
             dag_params = copy(dag.params) or {}
 
-        params = kwargs.get('params', {})
+        params = kwargs.get('params', {}) or {}
```

Comment:
This is correct, but looks odd. This is covered by a test right?

Reply:
https://github.com/apache/airflow/pull/7092/files#diff-beca6224080086e983c4dd9649fca3feR416 would fail if someone removes `or {}` so yes it is covered. 

I can add a separate test in `test/utils/test_decorator.py` if you think that would be good

Reply:
Think we're good